src/main/groovy/com/google/appengine/AppEnginePluginExtension.groovy

```diff
@@ -26,7 +26,10 @@ import org.gradle.api.Project
  * @author Benjamin Muschko
  */
 class AppEnginePluginExtension {
-    String httpAddress = "localhost"
+    String httpAddress =
```

Comment:
Can we use the elvis operator here from groovy?
http://docs.groovy-lang.org/latest/html/documentation/index.html#_elvis_operator

```
String httpAddress = System.getenv("DEV_APPSERVER_DEFAULT_ADDRESS") ?: "localhost"
```

Also not using standard google line lengths in this file, so feel free to go up to 120.